test/test_pysindy.py

```diff
@@ -490,7 +492,9 @@ def test_equations(data, capsys):
     model.print(precision=2)
 
     out, _ = capsys.readouterr()
+    pattern = re.compile(r"\(x0\)' = (.+)\n")
     assert len(out) > 0
+    assert pattern.match(out)
```

Comment:
Consider doing this without regexs, which are harder to read:
```suggestion
    assert len(out) > 0
    assert "(x0)' = " in out
```